takecare19/takecare19
# Delete a resource

### Testing instructions
* On the admin dashboard, click "Delete resource"
* Click Delete in the confirmation modal
* Confirm that the resource list has reloaded and that the deleted resource is gone

It'll probably be nice to add some sort of success state to show it was deleted
